cost/kubecost/cluster/kubecost_cluster_rightsizing_recommendations.pt

```diff
@@ -167,7 +239,7 @@ datasource "ds_cluster_sizing" do
   request do
     host $param_kubecost_host
     path "/model/savings/clusterSizing"
-    query "minNodeCount", $param_min_nodes
+    query "minNodeCount", join([$param_min_nodes])
```

Comment:
What is the purpose of the join statement here? Join is used to combine strings, but this only contains one string.

Reply:
we had inconvinents using only $param_min_nodes:

We initially used minNodeCount, but it seems the request did not include the correct parameter. We're unsure why this occurred.

<img width="533" alt="image" src="https://github.com/user-attachments/assets/16795497-0beb-4529-9143-fe95010820cc">


<img width="559" alt="image" src="https://github.com/user-attachments/assets/8854599b-6768-4417-9927-598442058b5a">

then using join statement appears that it's work well

Reply:
@joseluiszuflores maybe because the parameter value is a number, not a string? Though I would not expect that to have an impact on it not being able to interpret the query parameter.

Reply:
looks like our template can't have a number as param let me show  this scenario

I have changed the value by default  to 2 

<img width="478" alt="image" src="https://github.com/user-attachments/assets/6cf46026-9459-46bc-957e-86db7b6382c6" />

but the same behaviur on template,  we are received empty value

@XOmniverse  @nia-vf1

Reply:
Note: We updated our template to use the number 2 as a hardcoded value within the request. It seems to be working well.

template: 
<img width="580" alt="image" src="https://github.com/user-attachments/assets/ac3452d2-5d6c-4a7f-a6df-556f129b40c6" />

template applied 
<img width="502" alt="image" src="https://github.com/user-attachments/assets/2d4713c4-51e4-4030-847a-0475ba1fbbef" />


We think that  we use numbers like string, at least on request datastore.

example:  
* [name "AWS Savings Plan Recommendations"](https://github.com/flexera-public/policy_templates/blob/b72571abcfc6b13942d1736d30316dcc04fd797a/cost/aws/savings_plan/recommendations/aws_savings_plan_recommendations.pt#L352)
* [AWS Reserved Instances Recommendations](https://github.com/flexera-public/policy_templates/blob/b72571abcfc6b13942d1736d30316dcc04fd797a/cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt#L442)
* [Azure Rightsize SQL Managed Instance Storage](https://github.com/flexera-public/policy_templates/blob/b72571abcfc6b13942d1736d30316dcc04fd797a/cost/azure/rightsize_managed_sql_storage/azure_rightsize_managed_sql_storage.pt#L232)
* [Google Rightsize Cloud SQL Recommender](https://github.com/flexera-public/policy_templates/blob/b72571abcfc6b13942d1736d30316dcc04fd797a/cost/google/rightsize_cloudsql_recommendations/google_rightsize_cloudsql_recommendations.pt#L548)

link : Search's [link](https://github.com/search?q=repo%3Aflexera-public%2Fpolicy_templates++query+%22amount%22%2C+%221%22&type=code)